tests/postgres_tests/test_unaccent.py

```diff
@@ -42,6 +43,26 @@ def test_unaccent_chained(self):
             ordered=False
         )
 
+    def test_unaccent_chained_with_conforming_strings_off(self):
+        """
+        Escaping SQL should be valid when standard_conforming_strings is OFF
+        """
+        with connection.cursor() as cur:
+            cur.execute('SHOW standard_conforming_strings;')
+            conforming_strings_state = cur.fetchall()[0][0]
+            try:
+                if conforming_strings_state == 'on':
+                    cur.execute('SET standard_conforming_strings TO OFF;')
+                self.assertQuerysetEqual(
+                    self.Model.objects.filter(field__unaccent__endswith="éÖ"),
+                    ["àéÖ", "aeO"],
+                    transform=lambda instance: instance.field,
+                    ordered=False
+                )
+            finally:
```

Comment:
This is not necessary, the SET statement only lasts until the cursor is closed, which happens as soon as you exit the "with" block even in the event of an exception.